src/update-feeds-loops/contracts.ts

```diff
@@ -73,6 +73,10 @@ export const decodeUpdateParameters = (updateParameters: string): DecodedUpdateP
     ['uint256', 'int224', 'uint256'],
     updateParameters
   );
+  // 2 characters for the '0x' preamble + 3 parameters, 32 * 2 hexadecimals for 32 bytes each
+  if (updateParameters.length !== 2 + 3 * (32 * 2)) {
```

Comment:
The user (AirseekerRegistry owner) can specify `updateParameters` of arbitrary length, which is why it is important to check this imo. I check it after the decoding attempt because the error message for it will probably be more descriptive.

Reply:
Good point. We should definitely do this.